tests/scripts/run_performance.sh

```diff
@@ -74,14 +76,14 @@ run_device_perf_models() {
 
         env pytest models/demos/wormhole/mistral7b/tests -m $test_marker
 
-        env pytest "tests/ttnn/integration_tests/resnet/test_performance.py" -m $test_marker
-
         env pytest models/demos/resnet/tests -m $test_marker
     fi
 
     if [ "$tt_arch" == "wormhole_b0" ]; then
         env pytest models/demos/mamba/tests -m $test_marker
 
+        env WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest models/experimental/resnet/tests -m $test_marker
```

Comment:
same deal here

Reply:
There are grayskull skips in the test. I was copying the format of the other tests that have the wh arch yaml set. Is it fine to set that arch when on GS?

Reply:
I took a look and seems like the env var will be ignored unless we are actually using WH, so I've updated the cmds to be outside the if wh statements